src/models/Settings.js

```diff
@@ -54,6 +54,10 @@ export default Model.extend({
     // IDX API VERSION
     apiVersion: ['string', true, '1.0.0'],
 
+    // if ProxyIdxResponse is present in SIW config
```

Comment:
`attribute to hold proxy (fake) idx responses to render static pages without initiating idx pipeline`